components/mpas-ocean/src/mode_forward/mpas_ocn_time_integration_split.F

```diff
@@ -1101,31 +1112,32 @@ subroutine ocn_time_integrator_split(domain, dt)!{{{
                ! config_btr_gam1_velWt1=  0
                !    flux = normalBarotropicVelocityOld*H
 
+               !$omp parallel
+               !$omp do schedule(runtime)
+               do iEdge = 1, nEdgesAll
```

Comment:
@mark-petersen Thanks for chatting with me about the decomp issue. It ended up being this line where `nEdges` had to be `nEdgesAll` unlike the loop below.